config/build-cluster/cluster/200-cpu-limitrange.yaml

```diff
@@ -1,26 +0,0 @@
-# Copyright 2019 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: v1
-kind: LimitRange
-metadata:
-  name: cpu-limit-range
-  namespace: test-pods
-spec:
-  limits:
-    - default:
```

Comment:
I'd rather keep these sensible defaults

Reply:
Do you mean cpu or both?

Reply:
Both. It causes very weird intermittent problems when the default requests (.1cpu) are used by some jobs. It's just nice prevention-in-depth to keep.

Reply:
For lightweight jobs I actually would prefer it uses the default .1cpu as request, as well as any other pods such as janitor

Reply:
The other problem is that sidecar in each Prow job pod also requests these resources, which is pretty confusing

Reply:
So when we say each container requests 2 CPUs, a Prow job actually requests 4 CPUs

Reply:
Why don't we have resource requests on those?
I also think our janitors need a little more than .1 cpu, but we can reduce their quantity now that they've been fixed.

Reply:
Prow job sidecar is really lightweight, it shouldn't need 2 CPUs. If janitors need a little more than .1 cpu we can grant it there

Reply:
I asked someone to double-check, but looks like the sidecar is a problem. Could you boost the janitor CPU allocation slightly and reduce their number at the same time? The latter could be delayed if you want.

Reply:
done

Reply:
Ah, we had a misunderstanding. The sidecar shares the resources dedicated to the pod, so it doesn't hurt to keep the defaults.